index.html

```diff
@@ -128,54 +132,94 @@ <h3>Wootbit Software Develorper Agency</h3>
                 </div>
             </div>
         </section>
-<section id="about-me">
-    <div class="about-me-first">
-        <img id="top-right-img" src="./assets/svg/1-aboutme-mobile.svg" alt="">
-        <h2>About me</h2>
-        <p>  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            don’t hestiate to contact me.</p>
-            <a href="#">Get My Resume</a>  
-            <img id="bottom-left-img" src="./assets/svg/2-aboutme-mobile.svg" alt="">
-        </div>
-        <ul class="about-me-second">
-            <li class="second-card">
-                <div>
-                    <img src="./assets/png/icon-language.png" alt="">
-                </div>
-                <h3>Languages</h3>
-                <ul>
-                    <li>JavaScript</li>
-                    <li>PHP</li>
-                    <li>Go</li>
-                    <li>Ruby</li>
-                </ul>
-            </li>
-            <li class="second-card">
-                <div>
-                    <img src="./assets/png/icon-frameworks.png" alt="">
-                </div>
-                <h3>Frameworks</h3>
+        <section id="about-me">
+            <div class="about-me-first">
+                <img id="top-right-img" src="./assets/svg/1-aboutme-mobile.svg" alt="">
+                <h2>About me</h2>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
```

Comment:
- [OPTIONAL] Kindly use the font family, font size, and color same as the Figma template.